### DISCUSSION TITLE
beforBulkDestroy hook is not getting called

### DISCUSSION THREAD

**Discussion by doshishreyansh** [sequelize/sequelize#15414] [Help & Questions]
My hook **beforeBulkDestroy** is not getting called, can anyone help? Sequelize version: 6.25.5

Query:
`await BookingDetails.destroy(
					{
						where: { id: detailsDeleteIds },
						userId: req.user.id, //passing to hook
						individualHooks: true,
					},
					{
						transaction: t,
					}
				`);``

Hook inside model:
`hooks: {
				beforeDestroy: async (instance, options) => {
					await instance.update({ updatedBy: options.userId });
				},
				beforeBulkDestroy: async (instance, options) => {
					await instance.update({ updatedBy: options.userId });
				},
			},`

**Comment by ephys** [comment]
Apologies, I converted this to a question too quickly

I couldn't replicate the issue. If you can replicate it in https://github.com/sequelize/sequelize-sscce, consider opening a bug report about it using our [bug report template](https://github.com/sequelize/sequelize/issues/new?assignees=&labels=&template=bug_report.md&title=)
